appveyor.yml

```diff
@@ -1,28 +1,35 @@
 clone_depth: 1
 version: '{branch}-{build}'
 image: Visual Studio 2019
+cache:
+  - '%LOCALAPPDATA%\NuGet\v3-cache -> appveyor.yml'
+
 dotnet_csproj:
   patch: true
   file: 'osu.Game\osu.Game.csproj' # Use wildcard when it's able to exclude Xamarin projects
   version: '0.0.{build}'
-cache:
-  - '%LOCALAPPDATA%\NuGet\v3-cache -> appveyor.yml'
+
 before_build:
-  - ps: dotnet --info # Useful when version mismatch between CI and local
-  - ps: nuget restore -verbosity quiet # Only nuget.exe knows both new (.NET Core) and old (Xamarin) projects
+  - cmd: dotnet --info # Useful when version mismatch between CI and local
```

Comment:
Was there a particular reason for the `ps` -> `cmd` switch? Both aren't great but powershell is positively less ancient.

Reply:
Mostly just consistency - both `appveyor_deploy.yml` and the `ppy/osu-difficulty-calculator` repo uses this.

Reply:
Sure, that's a fair call.